app-commons/src/main/scala/org/bitcoins/commons/jsonmodels/dlc/DLCMessage.scala

```diff
@@ -64,16 +63,30 @@ object DLCMessage {
     }
   }
 
-  case class ContractInfo(outcomeValueMap: Map[Sha256DigestBE, Satoshis])
+  case class ContractInfo(outcomeValueMap: Vector[(Sha256Digest, Satoshis)])
       extends NetworkElement
-      with MapWrapper[Sha256DigestBE, Satoshis] {
-    override def wrapped: Map[Sha256DigestBE, Satoshis] = outcomeValueMap
+      with SeqWrapper[(Sha256Digest, Satoshis)] {
+    override def wrapped: Vector[(Sha256Digest, Satoshis)] = outcomeValueMap
+
+    def keys: Vector[Sha256Digest] = outcomeValueMap.map(_._1)
+
+    def values: Vector[Satoshis] = outcomeValueMap.map(_._2)
```

Comment:
lazy val for all

Reply:
I think I'll leave this one as is at least until 2206 is merged and that stuff is brought down as this all changes and will cause conflicts